### ISSUE TITLE
Panic with termion backend

### ISSUE THREAD

**Issue by spacekookie** [gyscos/Cursive#185]
# Problem description

Switching my application from using ncurses to termion panics with a very unhelpful message

```
 ❤ (rayya) ~/P/c/fool> env RUST_BACKTRACE=1 cargo run                                                                                                                   (master|✔) 
    Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
     Running `target/debug/fool`
 
thread 'main' panicked  at 'Red color fragment (r = 6) is out of bound. Make sure r ≤ 5.', /home/spacekookie/.cargo/registry/src/github.com-1ecc6299db9ec823/termion-1.5.1/src/color.rs:86:9
  stack backtrace:

0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace  at /checkout/src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
1: std::sys_common::backtrace::print  at /checkout/src/libstd/sys_common/backtrace.rs:68
                                      at /checkout/src/libstd/sys_common/backtrace.rs:57

2: std::panicking::default_hook::{{closure}} at /checkout/src/libstd/panicking.rs:381
3: std::panicking::default_hook at /checkout/src/libstd/panicking.rs:397
4: std::panicking::rust_panic_with_hook
5: std::panicking::begin_panic at /checkout/src/libstd/panicking.rs:538
6: std::panicking::begin_panic_fmt
7: termion::color::AnsiValue::rgb at /home/spacekookie/.cargo/registry/src/github.com-1ecc6299db9ec823/termion-1.5.1/src/color.rs:86
8: cursive::backend::termion::with_color at /home/spacekookie/.cargo/git/checkouts/cursive-fa9a0f4802024d18/df97279/src/backend/termion.rs:291
9: cursive::backend::termion::Concrete::apply_colors at /home/spacekookie/.cargo/git/checkouts/cursive-fa9a0f4802024d18/df97279/src/backend/termion.rs:61

0: <cursive::backend::termion::Concrete as cursive::backend::Backend>::clear at /home/spacekookie/.cargo/git/checkouts/cursive-fa9a0f4802024d18/df97279/src/backend/termion.rs:213
                                                                             at /home/spacekookie/.cargo/git/checkouts/cursive-fa9a0f4802024d18/df97279/src/cursive.rs:174

2: cursive::cursive::Cursive::set_theme at /home/spacekookie/.cargo/git/checkouts/cursive-fa9a0f4802024d18/df97279/src/cursive.rs:167
3: fool::main at src/main.rs:217
4: std::rt::lang_start::{{closure}}
5: std::panicking::try::do_call at /checkout/src/libstd/rt.rs:59
                                at /checkout/src/libstd/panicking.rs:480
6: __rust_maybe_catch_panic at /checkout/src/libpanic_unwind/lib.rs:101
7: std::rt::lang_start_internal at /checkout/src/libstd/panicking.rs:459
                                at /checkout/src/libstd/panic.rs:365
                                at /checkout/src/libstd/rt.rs:58
8: std::rt::lang_start at /checkout/src/libstd/rt.rs:74
9: main

0: __libc_start_main
1: _start                     
```

The stacktrace was quite badly formatted, hope it's readable and useful

This is being tested on Linux with the stable rust channel

**Comment by gyscos** [comment]
Whooops! Apparently I forgot to parse the grayscale color range (232-255), and chaos ensued.

Should be fixed in a9d9239fac7cd17257030d66d59cb1259bbf26f2.

**Comment by spacekookie** [comment]
Awesome 👍  That fixed it
